src/Symfony/Bridge/Doctrine/Tests/ArgumentResolver/EntityValueResolverTest.php

```diff
@@ -251,9 +250,7 @@ public static function idsProvider(): iterable
         yield ['foo'];
     }
 
-    /**
-     * @group legacy
```

Comment:
should we keep a `legacy` group for our tests covering deprecated behaviors ? Our `deps=high` job is excluding this group when running tests for a version that is not part of the last major version (as the dependencies could be installed at the next major version already): https://github.com/symfony/symfony/blob/fe94cfbc026db3987fe6bb284c5a844db2cd7a93/.github/workflows/unit-tests.yml#L134-L135

Reply:
thinking about this again, I think this option was overkill and was leading to not testing some parts of the code in CI in our LTS branches. Many legacy tests we have are about testing deprecations of their own component, which does not need such exclusion. Tests relying on cross-package deprecations could be responsible for skipping themselves when needed.

Reply:
I was chatting with @xabbuh about this yesterday and I think it'd be better to keep the existing policy of always using the legacy group when also skipping the deprecations. This worked really nice for years, and the code that is "overskipped" is still run on other matric lines, so the coverage it still as good as desired.
The issue with your proposal @stof is that it would require us to *think* about each test case, which means increasing the maintenance overhead. Using a simple rule that doesn't need any advanced thoughts makes automation possible for example, and things easy to explain to contributors (a bot can do it).